web/src/settings_users.js

```diff
@@ -336,6 +335,7 @@ section.active.create_table = (active_users) => {
         get_item: people.get_by_user_id,
         modifier_html(item) {
             const info = human_info(item);
+            info.display_last_active_column = true;
```

Comment:
This can also be initiated above in `human_info` to false and set to true here, but not too sure if initiating is better in one place compared to other or not. This variable doesn't seem like human info and didn't feel like it belonged there, so kept it here for now.